qiskit/assembler/disassemble.py

```diff
@@ -71,6 +73,24 @@ def _disassemble_circuit(qobj) -> CircuitModule:
     return CircuitModule((_experiments_to_circuits(qobj), run_config, user_qobj_header))
 
 
+def _qobj_to_circuit_cals(qobj, pulse_lib, param_pulses):
+    qobj_cals = qobj.config.calibrations.to_dict()['gates']
+    converter = QobjToInstructionConverter(pulse_lib)
+
+    qc_cals = {}
+    for gate in qobj_cals:
+        config = (tuple(gate['qubits']), tuple(gate['params']))
+        cal = {config: pulse.Schedule()}
```

Comment:
This should probably take a `name`... although I'm not sure if this information is actually sent in the qobj. You could check by adding a calibration with a named schedule. Maybe we can just set it to the gate name, or something like "{gate}({params}) {qubits}". What do you think?